packages/compass-e2e-tests/helpers/compass.js

```diff
@@ -303,6 +305,38 @@ function addDebugger(app) {
   }
 }
 
+function wrapCommands(app) {
+  const proto = Object.getPrototypeOf(app.client);
+  const commands = Object.keys(proto).filter((key) => {
+    return typeof proto[key] === 'function' && !key.includes('.');
+  });
+
+  const wrapped = {};
+  for (const command of commands) {
+    wrapped[command] = async function (...args) {
+      console.log(`${command} ${args.join(', ')}`);
+      const stack = new Error(command).stack;
+      try {
+        return await app.client[command].call(app.client, ...args);
+      } catch (error) {
+        // Log how we got here, but still throw the original error
+        error.stack = `${error.stack}\nvia ${stripWrapped(stack)}`;
+        throw error;
+      }
+    };
+  }
+
+  return wrapped;
+}
+
+function stripWrapped(stack) {
+  const lines = stack.split('\n');
+  // This is the same every time and not very useful
+  return lines
+    .filter((line) => !line.startsWith('    at Object.wrapped.<computed>'))
+    .join('\n');
+}
```

Comment:
Maybe it would make sense in this particular case to remove every line that doesn't originate in the source of this workspace, or would it be too tricky to do? I am looking at your example above and seems like lines that are starting from `helpers/` or `tests/` are the most useful there and things like `at processTicksAndRejections (node:internal/process/task_queues:96:5)` can all go. What do you think?

Reply:
Good idea.

Reply:
```
     Error: element ("this-does-not-exist") still not visible after 5000ms
      at new WaitUntilTimeoutError (/Users/leroux.bodenstein/mongo/compass/node_modules/webdriverio/build/lib/utils/ErrorHandler.js:149:12)
      at /Users/leroux.bodenstein/mongo/compass/node_modules/webdriverio/build/lib/commands/waitUntil.js:29:19
  via Error: waitForVisible
      at Context.<anonymous> (tests/ttfq.test.js:20:18)
```